ebpf/tracer.go

```diff
@@ -136,16 +203,28 @@ func (t *Tracer) getConnections() ([]ConnectionStats, error) {
 	// Remove expired entries
 	t.removeEntries(mp, tcpMp, expired)
 
-	return active, nil
+	// Add the shortlived connections
+	t.Lock()
+	active = append(active, t.closedConns...)
+	t.closedConns = []ConnectionStats{}
```

Comment:
Can we do the clearing to `closedConns` elsewhere?  This makes it so repeated curls to nettracer.sock will yield different results

Reply:
👍  I updated the PR to mark the connections as collected when the first get occurs and to add a separate goroutine that would check for collected closed connections every 10s and clean them from the Tracer